config/locales/de.yml

```diff
@@ -8,3 +8,9 @@ de:
       default_set: Default-Set
       radar: Radar
       modify: Ändern
+  ms:
```

Comment:
Just use `marital_statuses` here, the shortform doesn't seem necessary.
Reminds me too much of Microsoft anyways :grin: